src/configs/strings.ts

```diff
@@ -1,5 +1,5 @@
 export const GLOBAL_STRINGS = {
-  unassignComment: "Releasing the bounty back to dev pool because the allocated duration already ended!",
+  unassignComment: "Releasing the task due to lack of updates.",
```

Comment:
I removed 'bounty' from the terminology everywhere to make this more widely appealing.